content/packages/vine-lucid.yml

```diff
@@ -0,0 +1,19 @@
+name: '@akago/vine-lucid'
+description: Generate a vine schema based on a lucid model definition
+repo: akago-dev/vine-lucid
+npm: '@akago/vine-lucid'
+icon: ''
+github: https://github.com/akago-dev/vine-lucid
```

Comment:
404, link is broken

Reply:
Forgot to switch repository to public, sorry. Should be fine by now.